dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -45,7 +45,14 @@
  */
 class ClassDumper {
 
-  private static List<FullyQualifiedMethodSignature> listMethodReferences(JavaClass javaClass) {
+  /**
+   *  Lists all method references from the Java class. The output corresponds to
```

Comment:
"from the Java class": is this methods defined by the Java class or methods called by the Java class?

Reply:
The output list includes both (internal) methods defined in the file and (external) methods called by the class.

Updated source code comment.